src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkRepositoryContext.java

```diff
@@ -392,6 +392,84 @@ public boolean delete(Object pathObject, StarlarkThread thread)
     }
   }
 
+  @StarlarkMethod(
+      name = "rename",
+      doc =
+          """
+          Renames the file or directory from <code>src</code> to <code>dst</code>. \
```

Comment:
Can you please clarify that both paths should locate under the repository?

Reply:
Done.